test/test_dataloader.py

```diff
@@ -1484,8 +1484,8 @@ def check_len(dl, expected):
         check_len(DataLoader(self.dataset, batch_size=2), 50)
         check_len(DataLoader(self.dataset, batch_size=3), 34)
 
+    # FIXME: conversion from np.float64 to torch.DoubleTensor does not work
```

Comment:
I think my changes in PR https://github.com/pytorch/pytorch/pull/30486 will fix it, is there an issue filed somewhere tracking this bug?

Reply:
@nairbv cool! I couldn't find an issue for this, although I thought we had one filed. If I come across it I'll let you know.